### PR TITLE
Add .load command

### PR DESCRIPTION
**Pull request by zmwangx**
## Description

Added `.load` command (ref: `sqlite3 <<< .help | grep .load`) as discussed in #47.

## Checklist

- [x] I've added this contribution to the `CHANGELOG` file.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

File: litecli/packages/special/dbcommands.py [hunk 1]
```diff
@@ -149,3 +149,18 @@ def status(cur, **_):
...
+
+@special_command(
+    ".load",
+    ".load path",
+    "Load an extension library.",
+    arg_type=PARSED_QUERY,
+    case_sensitive=True,
+)
+def load_extension(cur, arg, **_):
+    path = arg.strip('"').strip("'")
```
Comment (meeuw) [inline litecli/packages/special/dbcommands.py]: Why did you add this? Wouldn't it be more appropriate to use shlex.split?

**Comment by meeuw** [comment]
Very useful! Please have a look at the review, I'd love to merge this.

File: litecli/packages/special/dbcommands.py (hunk 1 above)
Comment (zmwangx) [reply litecli/packages/special/dbcommands.py]: I used this because we're only dealing with a single argument, and bad inputs don't cause any harm, so this should be enough. ([Initially](https://github.com/dbcli/litecli/issues/47) I made sure quotes were matching, but threw it out when I saw `litecli.packages.special.main:show_keyword_help` getting by with [exactly this](https://github.com/dbcli/litecli/blob/3b942768b5797d0caf51f8e4f4cbcae1cdc7dfc4/litecli/packages/special/main.py#L146).) Argument handling elsewhere is pretty sloppy as well, e.g. `.schema`.

(Speaking of `show_keyword_help`, I just realized that code path is never touched because `help` doesn't accept any arguments.)

Nevertheless, I switched to `shlex` as requested.

By the way, a somewhat related question is why there's no tilde expansion here. That's because `sqlite3`'s `.load` doesn't support tilde expansion either, and the value of tilde expansion in this case is pretty limited to begin with.

**Comment by meeuw** [comment]
It looks good to me, thanks a lot!

**Comment by zmwangx** [comment]
Thanks for the merge.
